test/src/test/java/org/corfudb/integration/StreamIT.java

```diff
@@ -24,14 +27,43 @@ public void loadProperties() throws Exception {
         corfuSingleNodePort = Integer.parseInt((String) PROPERTIES.get("corfuSingleNodePort"));
     }
 
-//    @Test
-    public void simpleStreamTest() throws Exception {
+    @Test
+    public void simpleStreamFailOverTest() throws Exception {
+
+        File corfuLog = Files.createTempDir();
+        corfuLog.deleteOnExit();
+
+        Process corfuServerProcess = new CorfuServerRunner()
+                .setHost(corfuSingleNodeHost)
+                .setPort(corfuSingleNodePort)
+                .setLogPath(corfuLog.getAbsolutePath())
+                .runServer();
+
+        assertThat(shutdownCorfuServer(corfuServerProcess)).isTrue();
+
+        corfuServerProcess = new CorfuServerRunner()
+                .setHost(corfuSingleNodeHost)
+                .setPort(corfuSingleNodePort)
+                .setLogPath(corfuLog.getAbsolutePath())
+                .runServer();
+
+        simpleStreamRun();
+        assertThat(shutdownCorfuServer(corfuServerProcess)).isTrue();
+    }
+
+    @Test
+    public void simpleStreamNoFailOverTest() throws Exception {
 
         Process corfuServerProcess = new CorfuServerRunner()
                 .setHost(corfuSingleNodeHost)
                 .setPort(corfuSingleNodePort)
                 .runServer();
 
+        simpleStreamRun();
+        assertThat(shutdownCorfuServer(corfuServerProcess)).isTrue();
+    }
+
+    private void simpleStreamRun() {
         CorfuRuntime rt = createDefaultRuntime();
```

Comment:
Could you also fix this.
use createRuntime(corfuSingleNodeHost + ":" + corfuSingleNodePort).